js/repl/src/PresetLoadingAnimation.js

```diff
@@ -0,0 +1,56 @@
+import { css } from 'glamor';
+import React from 'react';
+
+const PresetLoadingAnimation = () =>
+  <div className={styles.loadingAnimation}>
+    <div className={`${styles.loadingTick} ${styles.loadingTick1}`} />
+    <div className={`${styles.loadingTick} ${styles.loadingTick2}`} />
+    <div className={`${styles.loadingTick} ${styles.loadingTick3}`} />
+    <div className={`${styles.loadingTick} ${styles.loadingTick4}`} />
+    <div className={`${styles.loadingTick} ${styles.loadingTick5}`} />
+  </div>;
+
+export default PresetLoadingAnimation;
+
+const bounce = css.keyframes({
+  '0%': { transform: 'scaleY(0.25)' },
+  '40%': { transform: 'scaleY(0.75)' },
+  '80%': { transform: 'scaleY(0.25)' },
+  '100%': { transform: 'scaleY(0.25)' }
+});
+
+const styles = {
+  loadingAnimation: css({
```

Comment:
The new React site is built with Gatsby, which does SSR, and uses the [Glamor plug-in](https://www.gatsbyjs.org/packages/gatsby-plugin-glamor/) which is optimized for the server-rendering.